modules/installation-registry-storage-block-recreate-rollout.adoc

```diff
@@ -16,7 +16,10 @@ To allow the image registry to use block storage types such as vSphere Virtual M
 
 [IMPORTANT]
 ====
-Block storage volumes are not recommended for use with image registry on production clusters. An installation where the registry is configured on block storage is not highly available because the registry cannot have more than one replica.
+Block storage volumes are *supported but not recommended* for use with image
```

Comment:
I actually meant the asterisks in the BZ comment to just highlight my suggested wording change. I don't think it looks all that bad included here, though it does go against our style guidelines so I'd be inclined to remove the bolding. WDYT?

Reply:
I flip-flopped a bit, but removed the asterisks to be more in line with our style guidelines. Thanks!